infrastructure-pr-testing.markdown

```diff
@@ -38,18 +44,58 @@ doing:
 
     # Start the new 4 workers by doing.
     aurora update start build/mesosci/devel/aliphysics_github_ci/4-7 aurora/continuos-integration.aurora
+
     # Those workers do the same job as worker 2 and 3 in the previous configuration.
     # Once they are up and running, we can therefore safely redeploy 2 and 3.
     aurora update start build/mesosci/devel/aliphysics_github_ci/2-3 aurora/continuos-integration.aurora
+
     # The new 2 and 3 will do the job of the old 1, which we can now redeploy
     aurora update start build/mesosci/devel/aliphysics_github_ci/1 aurora/continuos-integration.aurora
+
     # Finally we restart 0
     aurora update start build/mesosci/devel/aliphysics_github_ci/0 aurora/continuos-integration.aurora
 
+## Restarting the builders
+
+In some cases, builders need restart. The Aurora command for restarting a builder does not require
+any `.aurora` file as an option, and the builder will be restarted as it was deployed.
+
+All our production builders are under the tree `build/mesosci/devel`, so if you want to restart the
+builder called `build_AliRoot_el6native` we would do:
+
+    aurora job restart build/mesosci/devel/build_AliRoot_el6native
+
 
 # Monitoring the builders
 
-Builders are monitored in Monalisa. In particular you can use aliendb9
-and look at the `github-pr-checker/github.com/github-api-calls` metric
-to know how many API calls are being done by the system.
+Builders are monitored in [Monalisa](http://alimonitor.cern.ch/display?page=github/combined).
+In particular you can use aliendb9 and look at the `github-pr-checker/github.com/github-api-calls`
+metric to know how many API calls are being done by the system.
+
+
+# Currently active builders
+
+This is the list of builders that should be up and running.
+
+## Production
+
+| Repository           | GitHub check name        | Aurora job               | # |
```

Comment:
We should probably create the table from the Aurora API / Configuration itself.

Reply:
Yes, or maybe the other way around. The problem (and the reason why this table's there) is that we have too many things there (plus @brinick's tests, some of them not committed if I recall correctly) and it's really hard to find which ones are important.

Probably, we should remove all the non-production stuff from the `continuous-integration.aurora` configuration and use a separate file for clarity.